rdrf/rdrf/templates/rdrf_cdes/radio_select.html

```diff
@@ -1,17 +1,22 @@
 {# Adapted from 'django/forms/widgets/multiple_input.html' #}
 {% with id=widget.attrs.id %}
-  <ul{% if id %} id="{{ id }}"{% endif %} class="container-fluid radio-select {% if widget.attrs.class %}{{ widget.attrs.class }}{% endif %}">
+  <div{% if id %} id="{{ id }}"{% endif %} class="py-2 {% if widget.attrs.class %}{{ widget.attrs.class }}{% endif %}">
     {% for group, options, index in widget.optgroups %}
       {% if group %}
```

Comment:
Can you dig further into this and find out if there's any CDE configuration where the group is used? Adam and I can't think of any.

Can you also check if this template is used anywhere else or if the widget is used anywhere outside of the clinical forms (e.g. admin pages?)

Adam thinks this could possibly be residual code from a copy paste of the radio from the admin template. 

(Apologies for speaking on your behalf @ScriptSmith )

Reply:
All good. You can see what I believe is the original source (with some minor formatting and readability changes by us) + how django 4 changes it here: https://github.com/django/django/pull/14517/files#diff-80a2cdda9322d85740b395f796a72b5b5eec2dc76b9610bbd1685750d2135b0a

Reply:
If we were to keep the grouping, I'd be inclined to suggest some small changes to improve accessibility based on the WAI: https://www.w3.org/WAI/tutorials/forms/grouping/#associating-related-controls-with-fieldset
Mainly using appropriate attributes to better mark up the grouping and it's associated "label" without using a `<label/>` (which should be reserved for use where there is an associated input.)